Formula/daemontools.rb

```diff
@@ -22,6 +22,30 @@ def install
     end
   end
 
+  plist_options :startup => true
+
+  def plist; <<~EOS
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+    <dict>
+      <key>Label</key>
+      <string>#{plist_name}</string>
+      <key>ProgramArguments</key>
+      <array>
+        <string>sh</string>
+        <string>-c</string>
+        <string>mkdir -p /service &amp;&amp; exec #{opt_bin}/svscanboot</string>
```

Comment:
So this will create a directory in the root directory of the drive ?
Why would we need to do that and why not use one of the more common directories used for "shared"  information ?

Reply:
And what's stored in the directory, just runtime information ?

Reply:
Although it is strange, it is a contract of daemontools, so we can not change the placement location. For more information, you can see daemontools source codes.